### ISSUE TITLE
`<amp-lightbox-gallery>` zoom doesn't work on firefox

### ISSUE THREAD

**Issue by cathyxz** [ampproject/amphtml#14553]
Double tap, pinch, and tap zoom don't work on FF Android / FF iOS. Animation also contains visible flicker at end. 
Windows: lightbox doesn't open on FF Windows.

**Comment by ampprojectbot** [comment]
This is a high priority issue but it hasn't been updated in awhile. @cathyxz Do you have any updates?

**Comment by cathyxz** [comment]
Windows FF doesn't open bug was a fluke due to heroku CORS, fixed by https://github.com/ampproject/amphtml/pull/13886. The zoom issue on FF android is still present. Dropping to P2.

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @cathyxz Do you have any updates?

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @cathyxz Do you have any updates?

**Comment by sparhami** [comment]
Just tested this out on Firefox Android and I was able to double tap, then pinch to zoom on https://preview.amp.dev/documentation/examples/components/amp-lightbox-gallery. Closing this since there does not appear to be an issue with this currently. We did make a few changes for the double tap gesture, they may have fixed this as a side effect.
